bluer/src/lib.rs

```diff
@@ -1,4 +1,6 @@
 #![cfg_attr(docsrs, feature(doc_cfg))]
+#![cfg_attr(feature = "mesh", feature(type_alias_impl_trait))]
+#![cfg_attr(feature = "mesh", feature(associated_type_defaults))]
```

Comment:
BlueR targets stable Rust and as such cannot use nightly features.

Reply:
Turns out these were not necessary for `btmesh-common` and `btmesh-models` modules we're using.